examples/optimize_criteo.ipynb

```diff
@@ -19,7 +36,7 @@
     "df_valid = 'valid/'\n",
     "df_test = 'test/'\n",
     "\n",
-    "train_set = [data_path + df_train + x for x in os.listdir(data_path + df_train) if x.startswith(\"day_\") and not x.endswith(\"parquet\")] \n",
+    "train_set = [data_path + df_train + x for x in os.listdir(data_path + df_train) if x.startswith(\"day_\") and not x.endswith(\"parquet\")][:1] \n",
```

Comment:
Do you mean for this to only apply to one file?

Reply:
good call